src/index.ts

```diff
@@ -0,0 +1,5 @@
+export {MemoryServer} from './memory/server/MemoryServer';
+export * from './memory/client/MemoryBrowser';
+export * from './memory/client/MessageBroker';
+export * from './memory/client/index'
+export * from './extension'; 
```

Comment:
Is this still useful to you @alicetrifu given what you mentioned about exporting from `src/extension.ts`?

Reply:
It is. I need the classes from "./memory/client", in order to properly work. Regarding the src/extension.ts , I needed for some reason to do the export here as well, because the class was not recognized as exported until was on both places.

Reply:
How do you consume this package?

If you consume it as a regular npm package, then exporting from `src/extension.ts` is enough as it is the exposed script when doing `require('cdt-gdb-vscode')`.

This `src/index.ts` file is only ever useful if you do something like `require('cdt-gdb-vscode/out')`.

Just trying to straighten the setup here because it looks odd to export the same things from two separate places.

Reply:
@paul-marechal indeed, you are right. I've edited my extension and I do not need the src/index.ts. I'm doing the import by 'cdt-gdb-vscode'. I will do the push again without this file and as well the changes from tsconfig,json